packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -83,7 +83,7 @@ export class FungibleTokenTransferController implements ReactiveController {
   async init(env: Environment): Promise<void> {
     this.host.requestUpdate();
     this.env = env;
-    await this.config.init(1, env);
+    await this.config.init(1, this.env);
```

Comment:
why do we have chain id for mainnet hardcoded?

Reply:
because it's not used in our code and should be removed in SDK because it doesn't make sense 🤷🏻‍♂️